src/renderer/components/metrics/MetricPill.tsx

```diff
@@ -23,7 +23,9 @@ export const MetricPill: FC<IMetricPill> = (props: IMetricPill) => {
     >
       <Stack direction="horizontal" align="center" gap="none">
         <props.icon size={Size.XSMALL} className={props.color} />
-        {props.metric && <Text className="text-xxs px-1">{props.metric}</Text>}
+        {props.metric ? (
+          <Text className="text-xxs px-1">{props.metric}</Text>
+        ) : null}
```

Comment:
What's the reasoning behind this?

Reply:
https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&id=gitify-app_gitify&open=AZS1NATZtoWZECl0K8uP